portals/idn/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "moreInfoUrl": "https://idn.ceos.org/",
```

Comment:
the test failure can be fixed by updating line 60 on the `static/src/js/util/__tests__/portals.test.js` file. We only pull one portal in just to do a basic test; coincidentally this is the one that we use. If you update that value to this new `moreInfoUrl` it will pass then push that to this branch

Reply:
The issue has been resolved the portals.test.js has been updated.